packages/components/src/DataList/components/DataListHeaderTile/DataListSortingArrows.css

```diff
@@ -16,3 +16,8 @@
 .sortIcon path.inactive {
   fill: var(--color-inactive--surface);
 }
+
+.sortIcon:hover path {
+  filter: var(--color-interactive--subtle);
```

Comment:
This needs to be `fill:` instead of `filter:`, then we should be good to go!

Reply:
```suggestion
  fill: var(--color-interactive--subtle);
```

Reply:
Ah shoot, good catch, totally missed fill vs filter. Fixed!